vividus/src/test/resources/org/vividus/test-spring.xml

```diff
@@ -20,6 +20,8 @@
 
     <bean class="org.jbehave.core.steps.ParameterConverters.FluentEnumConverter" />
 
+    <bean class="org.vividus.converter.FluentTrimmedEnumConverter" />
```

Comment:
why is it needed?

Reply:
in other case I faced with issue
```
> Task :vividus:test
OpenJDK 64-Bit Server VM warning: Sharing is only supported for boot loader classes because bootstrap classpath has been appended

SpringStepFactoryIntegrationTests > customStepsInitializationTest() FAILED
    java.lang.IllegalStateException at DefaultCacheAwareContextLoaderDelegate.java:98
        Caused by: org.springframework.beans.factory.BeanCreationException at BeanDefinitionValueResolver.java:389
            Caused by: org.springframework.beans.factory.UnsatisfiedDependencyException at AutowiredAnnotationBeanPostProcessor.java:768
                Caused by: org.springframework.beans.factory.UnsatisfiedDependencyException at ConstructorResolver.java:800
                    Caused by: org.springframework.beans.factory.NoSuchBeanDefinitionException at DefaultListableBeanFactory.java:1801

620 tests completed, 1 failed
```
```
Caused by: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'SORTING' defined in class path resource [org/vividus/test-spring.xml]: Unsatisfied dependency expressed through constructor parameter 0; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type 'org.vividus.converter.FluentTrimmedEnumConverter' available: expected at least 1 bean which qualifies as autowire candidate. Dependency annotations: {}
	at app//org.springframework.beans.factory.support.ConstructorResolver.createArgumentArray(ConstructorResolver.java:800)
	at app//org.springframework.beans.factory.support.ConstructorResolver.autowireConstructor(ConstructorResolver.java:229)...
```

Reply:
`<bean class="org.jbehave.core.steps.ParameterConverters.FluentEnumConverter" />`
was deleted